test/Data.UnitTests/Detection/DetectionFactoryTests.cs

```diff
@@ -26,17 +29,32 @@ public void Factory_ShouldConvert_Entity()
             _model.ShouldBeEquivalentTo(outputDetection);
         }
 
+        [Test]
+        public void Factory_shouldConvert_MCreateDetection()
```

Comment:
Should we use the capital letter for the `_shouldConvert_` part?

Reply:
Whould the name `Factory_ShouldConvert_Input` be more relevant here?

Reply:
Done